.editorconfig

```diff
@@ -0,0 +1,55 @@
+# EditorConfig is awesome: https://EditorConfig.org
+# If you are using VS Code consider using the following extension
+# https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
+
+# The option max_line_length is only supported by some editors. Please read the official documentation.
+# https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[Makefile*]
+indent_style = tab
+indent_size = 8
+
+[{*.json,*.yml,*.yaml}]
+indent_style = space
+indent_size = 2
+
+[*.md]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = false
+
+[*.html]
+indent_style = tab
+
+[{*.css,*.sass,*.scss}]
+indent_style = tab
+
+[*.js]
+indent_style = space
+indent_size = 2
+
+[{**.min.js,**.min.css}]
```

Comment:
I was supposed to write these as `[**.min.{js,css}]`, i.e., only have the chagned part in `{}`, though I didn't yet do that and I also didn't have time to test that it works too. If you can do that, feel free to update both repos.

Reply:
I did the changes and I tested it in VS Code. I also added brackets to other relevant parts of the file. if these changes get accepted, I can make the modifications in the https://github.com/apluslms/a-plus/blob/master/.editorconfig